lib/web_ui/dev/test_platform.dart

```diff
@@ -841,24 +857,26 @@ class BrowserManager {
             suiteChannel,
             message);
 
-        final String sourceMapFileName =
-            '${p.basename(path)}.browser_test.dart.js.map';
-        final String pathToTest = p.dirname(path);
-
-        final String mapPath = p.join(env.environment.webUiRootDir.path,
-            'build', pathToTest, sourceMapFileName);
-
-        final Map<String, Uri> packageMap = <String, Uri>{
-          for (Package p in packageConfig.packages) p.name: p.packageUriRoot
-        };
-        final JSStackTraceMapper mapper = JSStackTraceMapper(
-          await File(mapPath).readAsString(),
-          mapUrl: p.toUri(mapPath),
-          packageMap: packageMap,
-          sdkRoot: p.toUri(sdkDir),
-        );
+        if (!_wasm) {
```

Comment:
Either this +
```
} else {
          controller!.channel('test.browser.mapper').sink.add(null);
}
```

or:
```
if (wasm) {
          controller!.channel('test.browser.mapper').sink.add(null);
} else { ... }
```

Also probably need a TODO here.

Reply:
Awesome, thank you.